qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -199,7 +201,13 @@ def setUp(self):
 
         self.configs_set = set()
 
+        log.info('Finished running CephFSTestCase.setUp().')
```

Comment:
What's the rational for printing the maps twice in setup and tearDown? Printing it once isn't sufficient?

Reply:
It would make it easier to investigate when an issue occurs not in the test methods but in these methods.